app/models/schemas.py

```diff
@@ -24,6 +26,16 @@ class Parent(BaseModel):
         regex=r'^[.A-Za-z0-9_-]*$'
     )
 
+    schedule_ruleset: ScheduleRuleset = Schema(
```

Comment:
Is `Parent` really `Zone`?  Why does zone have a schedule?

Reply:
May want an `AnnualSchedule` type that is the union of `ScheduleRuleset` and `ScheduleFixedInterval` that can be referenced wherever either type can be used?

Reply:
I will be clubbing the schedules with the load definitions in the program set. I could add an `AnnualSchedule` there instead of `Union[ScheduleRuleset, ScheduleFixedInterval]`.